src/Utilities/universalfallback.jl

```diff
@@ -0,0 +1,109 @@
+"""
+    UniversalFallback
+
+The `UniversalFallback` can be applied on a `ModelLike` `model` to create the model `UniversalFallback(model)` supporting *any* constraints and attributes.
```

Comment:
Does this actually support any constraint? It looks like it just forwards all constraints to the model.

Reply:
Yes, currently I have only attributes fallback but I was planning to add constraints too to fix https://github.com/JuliaOpt/JuMP.jl/issues/1152

Reply:
Having constraints working will be great :)
Just update the comment so it accurately describes the code that exists right now.